### PR TITLE
Update nginx

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *Manual modification was required to use onlyoffice*

## Solution
- *Apply [example nginx configuration](https://docs.nextcloud.com/server/18/admin_manual/installation/nginx.html)*
- *rewrite `documentserver_community` with `index.php` to be able to use onlyoffice right after installing it*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/nextcloud_ynh%20PR305/)  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by JimboJoe** [comment]
Let's merge in 3 days.

File: conf/nginx.conf [hunk 1]
```diff
@@ -69,15 +69,15 @@ location ^~ __PATH__/ {
     access_log off;
   }
 
-  location ~ ^__PATH__/(?:build|tests|config|lib|3rdparty|templates|data)/ {
+  location ~ ^\__PATH__\/(?:build|tests|config|lib|3rdparty|templates|data)\/ {
```
Comment (alexAubin) [inline conf/nginx.conf:72]: I'm randomly jumping in but won't this trigger weird issues considering that the nginx helper replaces `__PATH__` and `__PATH__/` by different thing in the case of installation on a domain root ? Or is it in fact on purpose ?

Comment (JimboJoe) [reply conf/nginx.conf:72]: but kay0u it was tested... was it? :sweat_smile:

Comment (kay0u) [reply conf/nginx.conf:72]: https://github.com/YunoHost/yunohost/blob/051d14a945ef00317cf79061389fb15b0fee951b/data/helpers.d/nginx#L35-L36

What kind of different thing? I tested it, it works, but I have to admit that I just copy pasta the current example, and I'm not sure if we should keep these `\`

**Pull request merged**